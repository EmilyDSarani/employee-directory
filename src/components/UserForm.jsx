// import { useState } from 'react';
// import { authForm } from '../../hooks/authForm';

//will need to use the formState and handleform change that has been used
//do I need to pass in error?
export default function UserForm() {
   
    // const { formState, handleFormChange  } = authForm({email:'', password:''});
    // const [loading, isLoading] = useState();



    //need a handleSubmit that takes email and password from formstate

    //try-catch to implement the password length

    //fieldset, legend, form, lebel, input, magic

    return (
    <>
        <fieldset>
            <legend>-insert Sign/Login name change magic here-</legend>
                <form>
                    <label htmlFor="email">Email </label>
                        <input
                        id="email"
                        type="email"
                        name="email"
                        // value={formState.email}
                        // insert onChange eventually
                        />
                    <label htmlFor="password">Password
                    </label>
                        <input
                        id="password"
                        type="password"
                        name="password"
                        // value={formState.password}
                        // insert onChange eventually
                        />
                  <button type="submit"> 
                      -insert the same login/sign up magic name change here-
                  </button>
                </form>
        </fieldset>
    </>
    )
}
