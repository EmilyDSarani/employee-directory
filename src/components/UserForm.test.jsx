import { render } from '@testing-library/react'
import UserForm from "./UserForm"

it('should display a form for login stuff', () => {
    const { container } = render(<UserForm />);
    expect(container).toMatchSnapshot();
});