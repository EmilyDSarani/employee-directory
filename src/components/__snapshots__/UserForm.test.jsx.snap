// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display a form for login stuff 1`] = `
<div>
  <fieldset>
    <legend>
      Auth
    </legend>
    <form>
      <label
        for="email"
      >
        Email 
      </label>
      <input
        id="email"
        name="email"
        type="email"
        value=""
      />
      <label
        for="password"
      >
        Password
      </label>
      <input
        id="password"
        name="password"
        type="password"
        value=""
      />
      <button
        type="submit"
      >
        Auth
      </button>
    </form>
  </fieldset>
</div>
`;
